library/core/src/any.rs

```diff
@@ -1220,15 +1223,16 @@ mod tags {
     /// A few example implementations for type-driven tags can be found in this
     /// module, although crates may also implement their own tags for more
     /// complex types with internal lifetimes.
-    pub trait Type<'a>: Sized + 'static {
+    #[unstable(feature = "error_generic_member_access", issue = "99301")]
```

Comment:
This is not a public API and shouldn't have a stability attribute.

Reply:
Without it I get the following compile error:

```
error: trait has missing stability attribute
    --> library/core/src/any.rs:1226:5
     |
1226 | /     pub(crate) trait Type<'a>: Sized + 'static {
1227 | |         /// The type of values which may be tagged by this tag for the given
1228 | |         /// lifetime.
1229 | |         type Reified: 'a;
1230 | |     }
     | |_____^

error: could not compile `core` (lib) due to previous error
```

Reply:
I don't think this trait needs to be `pub(crate)` since all of the users are in this module. If that still doesn't work then  mark it with a comment so that it doesn't accidentally get stabilized along with the feature.

Reply:
There are several places outside the `tags` module but inside the `error` module where it is referenced so I'll leave the comment.

Reply:
Okay now that it's moved into the `error` module I _don't_ get that "trait has missing stability attribute" error. I hadn't tried removing it after moving the `tags` module and `Request` type into the `error` module. I'll try removing the stability attribute from other items in the `tags` module also.